### PR TITLE
Warn users they cannot search by email address [OSF-5639] 

### PR DESCRIPTION
**Pull request by zachjanicki**
## Purpose

Currently users can search when adding contributors by email address. They can then invite a user, who may already exist in the database by the email address entered, which can cause confusion when it says they already have an account. These changes will no longer enable users to search by email address.

## Changes

If a user attempts to search an email address in the add contributors page, it will show a message that searching by email address is not allowed, and it will not give them the option to add that email address as an unregistered contributor.

### BEFORE:

image

### AFTER:
image

## Side effects

None.

## Ticket

https://openscience.atlassian.net/browse/OSF-5639

### CODE REVIEW

**Comment by brianjgeiger** [comment]
I'm guessing your before and after photos are mixed up?

**Comment by zachjanicki** [comment]
Ah yes they were. It is corrected.

File: website/static/js/contribAdder.js [hunk 1]
```diff
@@ -96,10 +96,17 @@ AddContributorViewModel = oop.extend(Paginator, {
         self.totalPages = ko.observable(0);
         self.childrenToChange = ko.observableArray();
 
+        self.emailSearch = ko.pureComputed(function () {
+            var emailRegex = new RegExp('[a-zA-Z0-9]+@[a-zA-Z0-9]+\.[a-z]+');
```
Comment (brianjgeiger) [inline website/static/js/contribAdder.js]: We would never use this for real email address validation, but it should be fine for the purpose.

Comment (brianjgeiger) [inline website/static/js/contribAdder.js]: No, wait, I don't think that would work for some cases that QA will almost certainly use. (i.e. nici+testuser1@cos.io) Well maybe.

Still, take a look at http://stackoverflow.com/a/1373724 and maybe do the practical one (include a comment to the link to that answer so people can see where it came from and why you used it)

**Comment by brianjgeiger** [comment]
Okay, one small correction, and please add updated screenshots showing each of the conditions you've modified. Then I'll have Sara verify and we'll be done. 🐧

**Comment by mfraezz** [comment]
Other than the small UI change, LGTM. This should be ready soon 🌵

**Pull request closed without merging**